content/components/action-bar.mdx

```diff
@@ -1,84 +1,89 @@
 ---
 title: Action bar
 status: Experimental
-description: An action bar contains a collection of horizontally aligned icon buttons.
+description: An action bar contains a collection of horizontally aligned actions.
 ---
 
 import {Box, Text} from '@primer/react'
 
-## Overview
-
-Use an action bar to render multiple icon buttons in a row. Buttons can be split into groups by adding a divider. When there is not enough space, buttons that don't fit will be added to an overflow menu.
+<img
+  width="960"
+  alt=""
+  src="https://user-images.githubusercontent.com/378023/197716033-86fdac40-2b14-438c-a712-b003b76dabd3.png"
+/>
 
 ## Anatomy
 
+And action bar is mostly a wrapper for different kind of actions. It makes sure alliment and responsiveness works without much configurations.
+
 <img
   width="960"
   alt="A diagram of an action bar with a few buttons, a divider and at the end a button that opens an overflow menu."
```

Comment:
```suggestion
  alt="A diagram of an action bar with a few buttons, a divider and a button that opens an overflow menu at the end."
```

Reply:
Wouldn't this suggest that the "overflow menu" is opened at the end? Maybe I'll just remove `at the end` altogether to not cause confusion.